### ISSUE TITLE
Next/Previous change navigation hotkeys don't work

### ISSUE THREAD

**Issue by andalerom** [comod/git-scope-pro#15]
Hotkeys for "Next/Previous change" navigation in file don't work with GitScope plugin enabled:  
- `Ctrl+Alt+Shift+Down` 
- `Ctrl+Alt+Shift+Up`

![image](https://user-images.githubusercontent.com/17616026/124952366-d8722f00-e01c-11eb-9435-9dcb31390745.png)

However, when the change-popup is displayed, the hotkeys **do** work (same as ⬆️/⬇️ buttons on the popup itself):
![image](https://user-images.githubusercontent.com/17616026/124954306-9ba73780-e01e-11eb-86e4-c439befc74ad.png)

**Comment by comod** [comment]
Thanks for reportin. I cannot reproduce this! do you still have this issue?

**Comment by andalerom** [comment]
Yep. Ctrl+Alt+Shift+Up/Down does not work in all recent IDEA versions.

**Comment by vassudanagunta** [comment]
I have also had this issue for many months, and only today discovered that Git Scope is the culprit.  See [this issue I opened with JetBrains](https://youtrack.jetbrains.com/issue/IDEA-275025), which I opened because I originally thought it was an JetBrains bug.

**Comment by comod** [comment]
i havent used this shortcuts before. they work as long as the rollback-toolbar is opened (what you see on the screenshot). Should they work even when the toolbar is not visible?

**Comment by comod** [comment]
For me all shortcuts works as expected with Git Scope enabled. Maybe it is a OS related problem!?

**Comment by vassudanagunta** [comment]
Mine is macOS Big Sur.

**Comment by comod** [comment]
I have now been able to recreate the problem. When the popup is open, all shortcuts work. Only when the popup is closed, the default behavior is affected. I think I have a solution in mind, but the effort of implementation is not proportional to the problem, especially because the whole implementation will have to be replaced due to deprecations..  There is also the Diff window which can be opened with CTRL+D (on any file in the diff window) and with which you can jump from change to change with F7.
